### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by dheerajkd30**

### CODE REVIEW

File: testnet/genesis.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"TBU" : "29 sep"}
```
Comment (bsrinivas8687) [inline testnet/genesis.json]: Create a new repository called `networks` for this

File: testnet/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+# comdex-test-1
```
Comment (bsrinivas8687) [inline testnet/README.md]: This should be under docs repository

File: logo.svg [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<svg width="120" height="25" xmlns="http://www.w3.org/2000/svg">
```
Comment (bsrinivas8687) [inline logo.svg:1]: Do we have any logo already hosted somewhere? If yes, remove this file

**Review by bsrinivas8687 — CHANGES_REQUESTED** [review]

File: logo.svg (hunk 3 above)
Comment (dheerajkd30) [reply logo.svg:1]: not hosted yet, once done will update.

**Review by bsrinivas8687 — APPROVED** [review]

**Pull request merged**